http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/ReactiveRestClientIT.java

```diff
@@ -60,6 +64,22 @@ public void shouldGetBookFromRestClientJson() {
         assertEquals("Title in Json: 123", response.jsonPath().getString("title"));
     }
 
+    @Test
+    public void shouldPostBook() {
+        Response response = app.given()
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .when()
+                .post("/client/book/search");
+        assertEquals(HttpStatus.SC_OK, response.statusCode());
+        Set<Book> books = new HashSet<>(response.jsonPath().getList(".", Book.class));
```

Comment:
Is there a reason why not use list which is returned?

Reply:
I will update it to simplify the code to List.